requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2==2.5.4
+https://github.com/warner/python-tweetnacl/tarball/master
```

Comment:
Hmm, mozno radsej nejaky tag alebo konkretny hash?
(Gah, to neni na pypi? Skoda...)